### PR TITLE
fix: group permissions

### PR DESCRIPTION
**Pull request by vivzhang**
1. Description:

- Change group members and content permissions to view
- Fix typo in `SortOption`

1. Instructions for testing:

1. Closes Issues: #<number> (if appropriate)

1. [ ] Updated meaningful TSDoc to methods including Parameters and Returns, see [Documentation Guide](https://esri.github.io/hub-components/storybook/?path=/story/guides-documentation--page)

1. [ ] used semantic commit messages
  
1. [ ] PR title follows semantic commit format (**CRITICAL** if the title is not in a semantic format, the release automation will not run!)

1. [ ] updated `peerDependencies` as needed. **CRITICAL** our automated release system can **not** be counted on to update `peerDependencies` so we _must_ do it manually in our PRs when needed. See the [updating peerDependencies](/RELEASE.md#Updating-peerDependencies) section of the release instructions for more details.

### CODE REVIEW

File: packages/common/src/groups/_internal/GroupBusinessRules.ts [hunk 1]
```diff
@@ -113,10 +113,10 @@ export const GroupPermissionPolicies: IPermissionPolicy[] = [
   },
   {
     permission: "hub:group:workspace:content",
-    dependencies: ["hub:group:edit"],
+    dependencies: ["hub:group:view"],
```
Comment (sonofflynn89) [inline packages/common/src/groups/_internal/GroupBusinessRules.ts:116]: Let's double check with Thomas on this change.

Comment (vivzhang) [reply packages/common/src/groups/_internal/GroupBusinessRules.ts:116]: This is for the entity group content, not the entity content :)

Comment (sonofflynn89) [reply packages/common/src/groups/_internal/GroupBusinessRules.ts:116]: Ahhhh my bad. Thanks for the clarification!

**Review by sonofflynn89 — APPROVED** [review]
Thanks Viv! 🙏

**Pull request merged**